example/animation.rb

```diff
@@ -25,11 +25,13 @@ def out.tty?; true; end
   $stdout.print "\e[0J"
   $stdout.flush
 
-  n = lines.count
-  $stdout.print "\e[#{n}F"
-  shift = (shift + M) % N
-
   sleep 0.2
+
+  if continue
+    n = lines.count
+    $stdout.print "\e[#{n}F"
+    shift = (shift + M) % N
+  end
```

Comment:
I see. I should have understood your idea of not erasing the plot on exit.
Sorry.